charts/pmm/templates/NOTES.txt

```diff
@@ -0,0 +1,16 @@
+1. Get the application URL by running these commands:
+{{- if contains "NodePort" .Values.service.type }}
+  export NODE_PORT=$(kubectl get --namespace {{ .Release.Namespace }} -o jsonpath="{.spec.ports[0].nodePort}" services {{ include "pmm.fullname" . }})
```

Comment:
this {{ include "pmm.fullname" . }} is resolving to `pmm` when the actual (default) service name is `monitoring-service`

Reply:
fixed